Assets/Scripts/Nodes/BuddyNodes.cs

```diff
@@ -47,6 +49,7 @@ public override NodeStatus Tick()
 		offset.z = offset.y;
 		offset.y = 0;
 		baseInfo.destination = transform.position + offset;
+		controller.moveDirection = Vector3.zero;
```

Comment:
I'm actually not sure if this is necessary. I think it is but I don't know for certain.